concordium-cis2/src/lib.rs

```diff
@@ -1108,6 +1068,7 @@ impl From<AccountAddress> for Receiver {
 #[derive(Debug, Serialize, Clone)]
 pub struct AdditionalData(#[concordium(size_length = 2)] Vec<u8>);
 
+// Implemented manually to hide the newtype wrapper.
```

Comment:
We should use `transparent` here.

Reply:
This PR is preceding the PR adding `transparent`, I solve this in https://github.com/Concordium/concordium-rust-smart-contracts/pull/297

Reply:
Ah 👍